src/graph/heterograph.h

```diff
@@ -249,6 +249,15 @@ class HeteroGraph : public BaseHeteroGraph {
    */
   void UnpinMemory_();
 
+  /**
+   * @brief Copy the current graph to pinned memory managed by
+   *     PyTorch CachingHostAllocator for each relation graph.
+   * @note If any of the underlying relation graphs are already pinned, the
+   *     function simply uses its original copy. If all pinned, it returns
+   *     itself.
```

Comment:
nit: gramma issue. it returns itself, the it means the function, but the return is the original graph.

from ChatGPT with manual tweak:
If any of the underlying relation graphs are already pinned, the function will utilize their existing copies. If all of them are pinned, the function will return the original input hetero-graph directly.

Reply:
Updated.